syft/pkg/cataloger/golang/scan_binary.go

```diff
@@ -6,12 +6,19 @@ import (
 	"io"
 	"runtime/debug"
 
+	version "github.com/kastenhq/goversion/version"
```

Comment:
nit: can be simplified
```suggestion
	"github.com/kastenhq/goversion/version"
```